### PR TITLE
docs: fix example folder names to be `deno-fresh-demo`

### PR DESCRIPTION
**Pull request by moonstripe**
Figured it'd make sense to be consistent.

### CODE REVIEW

File: docs/.DS_Store
Comment (iuioiua) [inline docs/.DS_Store:1]: Can you please delete this file?

**Review by iuioiua — CHANGES_REQUESTED** [review]
Two other instances in the codebase mention `my-project` instead of `deno-fresh-demo`. Could you please adjust these too?

File: docs/.DS_Store
Comment (moonstripe) [reply docs/.DS_Store:1]: Done

**Comment by moonstripe** [comment]
> 

Where are these other mentions? Couldn't find them in the docs.

**Comment by iuioiua** [comment]
Try searching `my-project`. The file paths that include this are:
- [x] /www/routes/index.tsx
- [x] /www/routes/update.tsx

**Review by iuioiua — APPROVED** [review]
I applied my suggested fixes myself. Now, LGTM! Thank you.

**Pull request merged**